lib/stdlib/doc/src/erl_pp.xml

```diff
@@ -45,6 +45,12 @@
 
     <p>All functions can have an optional argument, which specifies a hook
       that is called if an attempt is made to print an unknown form.</p>
+
+    <p>Note that if the functions in this module are used to produce
+      Erlang source code from forms produced by the Erlang compiler, the
```

Comment:
```suggestion
    <p>Note that if the functions in this module are used to convert abstract
    code back to Erlang source code that can be compiled, the
```

(Other compilers than the Erlang compiler may produce variable names that are not syntactically correct.)

Reply:
I don't like the qualification of "source code that can be compiled", my experience with Bugerl is that most output can be compiled, it just doesn't have the same semantics as the input. I have reformulated it to talk about output that is semantically equivalent to the abstract code.